lib/browse_everything/driver/s3.rb

```diff
@@ -3,12 +3,17 @@
 module BrowseEverything
   module Driver
     class S3 < Base
-      DEFAULTS = { signed_url: true, region: 'us-east-1' }.freeze
-      CONFIG_KEYS = [:app_key, :app_secret, :bucket].freeze
+      DEFAULTS = { response_type: :signed_url }.freeze
+      RESPONSE_TYPES = [:signed_url, :public_url, :s3_uri].freeze
+      CONFIG_KEYS = [:bucket].freeze
 
       attr_reader :entries
 
       def initialize(config, *args)
+        if config.key?(:signed_url) && config.delete(:signed_url) == false
+          warn '[DEPRECATION] Amazon S3 driver: `:signed_url` is deprecated.  Please use `:response_type` instead.'
+          config[:response_type] = :public_url
```

Comment:
This isn't clear about how to get `signed_url` behavior. Unless wait, `config[:resonse_type] = :public_url` just means the same thing as `signed_url: true` used to mean?  I don't understand how public_url is a synonym for signed_url, but if that's all it is, that's fine.  The deprecation warning should maybe say "Please use `response_type: :public_url`" instead? Assuming that's the new synonym?

Reply:
Actually, from looking at code below, I believe this is _wrong_.  And the code is wrong. The equivalent to `signed_url: true` before looks like `response_type: :signed_url`, no?  If that is true, then this code breaks people who asked for `signed_url`, giving them a `public_url` before, which is not a `signed_url`.

Reply:
Please open up a new issue. This PR has been merged.